# # config.yaml

trajectory_node:
  
  ros__parameters:
    
    joint_names : ['LH_HFE', 'LH_KFE']
    
    topic :
      joint_state    : '/joint_states'
      high_rate_traj : '/high_rate_traj'
      low_rate_traj  : '/low_rate_traj'
    
    rate :
      high_rate : 200.0
      low_rate  : 50.0

    task :
      duration : 2.0
      wait_t   : 5.0
      pf       : [-0.25, 0.0]
    


fake_pd_node:
  
  ros__parameters:
    
    joint_names: ['LH_HFE', 'LH_KFE']
    
    topic:
      joint_state : '/joint_states'
      des_traj    : '/high_rate_traj'
      command     : "/joint_controller/command"
    
    rate: 200.0
    